src/main/kotlin/net/irisshaders/lilybot/extensions/util/Tags.kt

```diff
@@ -123,6 +144,12 @@ class Tags : Extension() {
 			}
 		}
 
+		/**
+		 * The command for deleting tags
```

Comment:
This should end in a period right? I'm surprised DeteKt didn't yell at you

Reply:
Me too